exonum-java-binding/core/src/main/java/com/exonum/binding/blockchain/Blockchain.java

```diff
@@ -239,4 +240,13 @@ public Block getLastBlock() {
   public StoredConfiguration getActualConfiguration() {
     return schema.getActualConfiguration();
   }
+
+  /**
+   * Returns a set of uncommitted (in-pool) transaction hashes; empty in case of no transactions.
+   *
+   * @see <a href="https://exonum.com/doc/version/0.10/advanced/consensus/specification/#pool-of-unconfirmed-transactions">Pool of Unconfirmed Transactions</a>
+   */
+  KeySetIndexProxy<HashCode> getPoolTransactionHashes() {
```

Comment:
I am not entirely happy with the name, what if it is named `getTransactionPool` and the documentation clarifies that a transaction pool is a set of hashes of uncommitted transaction messages?

Reply:
Is that correct that it is intended to be internal (hence package-private and, supposedly, invocation through reflection)? If it is not I would also add a warning that this pool represents the state as of the current snapshot, and its state constantly changes even between block commits.

Reply:
Yes, I was inclining to make it public